src/components/Calculator.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable react/button-has-type */
+import React, { Component } from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Please remove the `eslint-disable` line and try to fix it manually as this defies the purpose of using linters in the first place.